docs/ODK_migrated/DeletingAssertedSubClasses.md

```diff
@@ -0,0 +1,19 @@
+**NOTE** This documentation is incomplete, for now you may be better consulting the [GO Editor Docs](http://wiki.geneontology.org/index.php/Ontology_Editing_Guide)
```

Comment:
I think this belongs in discussion/explanation (clicking X feels like an obvious how to delete axiom, and the whole concept of this is letting the reasoner do the work over asserting stuff) - Will remove this once I find something that replaces this in an explanation doc

Reply:
moved